### PR TITLE
Pull request to test the "Build docker image" circle ci step

### PR DESCRIPTION
**Pull request by pdedumast**

### CODE REVIEW

**Comment by sebastientourbier** [comment]
Hi @pdedumast ,
It is indeed what I would have done... 🤔 
Let you know if I get another idea!

**Comment by sebastientourbier** [comment]
I just tried to remove some conda channels that might be problematic, not sure if there are needed thought. Let's see

File: .circleci/config.yml [hunk 1]
```diff
@@ -67,13 +67,13 @@ jobs:
             # Build main docker image
             ls -la .
             e=1 && for i in {1..5}; do
-              docker build --cache-from sebastientourbier/mialsuperresolutiontoolkit-ubuntu14.04 --rm=false --build-arg BUILD_DATE=`date -u +"%Y-%m-%dT%H:%M:%SZ"` --build-arg VCS_REF=$VCS_REF --build-arg VERSION="${CIRCLE_TAG:-$THISVERSION}" -t sebastientourbier/mialsuperresolutiontoolkit-ubuntu14.04 . \
+              docker build --cache-from sebastientourbier/mialsuperresolutiontoolkit-ubuntu14.04 --rm=false --build-arg BUILD_DATE=`date -u +"%Y-%m-%dT%H:%M:%SZ"` --build-arg VCS_REF=$VCS_REF --build-arg VERSION="${CIRCLE_TAG:-$THISVERSION}" --memory-swap -1 -t sebastientourbier/mialsuperresolutiontoolkit-ubuntu14.04 . \
```
Comment (sebastientourbier) [inline .circleci/config.yml:70]: It seems this solved the issue with docker build, no?

Comment (pdedumast) [reply .circleci/config.yml:70]: It did, yes! Problems for build-singularity now... 😩

Comment (pdedumast) [reply .circleci/config.yml:70]: Or maybe not in the end...

**Pull request closed without merging**